src/shared/platforms/android/androidcommons.cpp

```diff
@@ -61,8 +61,7 @@ bool AndroidCommons::verifySignature(const QByteArray& publicKey,
 // static
 bool AndroidCommons::shareText(const QString& text) {
   return (bool)QJniObject::callStaticMethod<jboolean>(
-      UTILS_CLASS, "sharePlainText",
-      "(Landroid/content/Context;Ljava/lang/String;)Z", getActivity().object(),
+      UTILS_CLASS, "sharePlainText", "(Ljava/lang/String;)Z",
```

Comment:
This is wrong. Shared plaintext should be in the shared android utils. Not in the VPN one. 

Can you move it there? And remove the UTILS macro on top.

Reply:
Also launch PlayStore should be in common utils.